common/api/core-frontend.api.md

```diff
@@ -9567,22 +9567,6 @@ export interface SelectReplaceEvent {
     type: SelectionSetEventType.Replace;
 }
 
-// @alpha

```

Comment:
did you get this chunk deletion from running `rush extracta-api`?
curious were there no csv changes along with this?

Reply:
Oh there's a command for that? I previously deleted it manually. Should I run `rush extract-api`?

Reply:
Is it normal to get all these changes?
![image](https://user-images.githubusercontent.com/51834103/176790744-be3fbfe0-8283-46ac-82d8-20acf359d294.png)

Reply:
are most of those changes just line endings?
whats up with that csv file with your name on it?